content/rocur/admin guide/index.en.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Admin guide"
+menuTitle: "Admin guide"
+weight: 60
+---
+
+## Account Admin  
+
+The account admin for the rotating curator account are members of the R-Ladies community that keeps @WeAreRLadies running. The account admin could be a team of 1-4 people who can rotate responsibilities from month-to-month. Being an account admin is an opportunity to connect with R-Ladies from all over the world while facilitating an online community resource.     
+
+## Primary Responsibilities  
+
+The responsibilities of the account admin are to:  
+* Coordinate and grant Twitter account access to curators.
+* Provide instructions to curators and answer questions as needed.  
+* Recruit R-Ladies to be curators for the account.  
+* Update and expand on the below procedures as needed.   
+
+## Admin Procedures  
+
+### When someone signs up to curate:    
+* Add them to the curator schedule as soon as possible so other potential curators know that week is unavailable.  
+* The curator should send an email to WeAre@rladies.org with a photo for the account  
+* When this photo is received, save it in the Curator Photos Folder.  
```

Comment:
Is this one some sort of shared drive?

Reply:
Yes! I have a shared google folder in the google drive of R-Ladies Chicago (I do not know if there is a better place for this at this time...)